### ISSUE TITLE
[Supporting the -Howdyai]: In resize mode(200%) “Howdy, Help, Logout, Dashboard, scripts” controls are not visible in the howdy page. 

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#5]
**User Experience**
Users who rely on AT will face difficulty and can get confused if at 200% scaling, all the controls are not visible properly, then the user will not be able to perform the actions on that control successfully.

**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://howdy.ai/
Browser: Edge

**Repro Steps**
Steps to Follow Install Slack:
Step 1: Open the URL: https://howdy.ai/
Step 2: Navigate to "@howdy for Slack" and select it.
Step 3: Navigate to "Create Workspace" and select it.
Step 4: Sign In with valid credentials to open slack account.
Step 5: Now go to Howdy Home page URL: https://howdy.ai/
Step 6: Repeat Step 2 again and select it.
Step 7: Navigate to "Install" button and select it.
Step 8: Select "Authorize" button and select it.
Step 9: Select "Turn your Boot On"
Step 10: Click on "Continue" button.
Step 11: "Howdy Dashboard" page gets opened.
Step 12: Navigate to "Launch Howdy in Slack".
Step 13: Navigate to "Dashboard" tab and select it.
Step 14: "Howdy Dashboard" page gets opened.
Step 15: Apply the 200%  resize mode 
Step 16: Observe the issue.

**Actual Result**
In resize mode (200%) “Howdy, Help,Logout,Dashboard,scripts”  controls are not  visible

**Expected Result** 
“Howdy, Help,Logout,Dashboard,scripts”  controls should be visible Resize mode 200% 

**Mas Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

 

![MAS1 4 4_Howdy](https://user-images.githubusercontent.com/48272005/54471461-57edeb80-47df-11e9-8f6a-457cc38db399.png)

**Comment by benbrown** [comment]
This was due to an unintended consequence of our "mobile friendly" menu, which was out of date.

This should now be fixed.

**Comment by v-gasikk** [comment]
Issue is partially Fixed.
Logout button is not visible in 200% Zoom mode.

![MAS1 4 4_Howdy_Logout](https://user-images.githubusercontent.com/48272005/55053394-0c123080-5082-11e9-84a5-5f63645dd5c1.png)

**Comment by benbrown** [comment]
The logout link is now visible.

**Comment by v-gasikk** [comment]
Verified. Issue is Fixed. Hence Closing the bug.
